layouts/default.vue

```diff
@@ -1,5 +1,18 @@
 <template>
-  <div>
-    <Nuxt />
+  <div class="container mx-auto m-0 p-2 h-screen max-w-4xl">
+    <div class="flex m-4 justify-around">
```

Comment:
I think explicitly setting max-width conflicts with the implicit width setting of the container class, see https://tailwindcss.com/docs/container. Can we remove one of these styles?

Reply:
the `max-w-4xl` is removed, see my last two commits.